### PR TITLE
Update http_cookies

### PR DESCRIPTION
**Pull request by tsellers-r7**
## Description
This PR updates `http_cookies` by:
- Adding missing examples
- Re-ordering some of the matches to improve metrics
- Removing unnecessary captures where the value doesn't change.

## Motivation and Context
Improving fingerprints

## How Has This Been Tested?
`rspec`

## Checklist:

- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/http_cookies.xml [hunk 1]
```diff
@@ -5,9 +5,49 @@
   servers.
   -->
 
+  <fingerprint pattern="^__cfd?uid=">
+    <description>CloudFlare web load balancer endpoint</description>
+    <example>__cfuid=1337</example>
+    <example>__cfduid=dd450f2431e1e611a61a15f68974de9a41618794671; expires=Wed, 19-May-21 01:11:11 GMT; path=/; domain=.foo.bar; HttpOnly; SameSite=Lax</example>
```
Comment (tsellers-r7) [inline xml/http_cookies.xml:11]: Fingerprint moved up due to frequency of observation.  Coverage for `__cfduid` added

File: xml/http_cookies.xml [hunk 2]
```diff
@@ -5,9 +5,49 @@
...
+    <example cookie="PHPSESSID">PHPSESSID=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0; path=/</example>
+    <example cookie="PHPSESSION">PHPSESSION=vt2ag6n7t6ngvlg8adk4860h46; path=/</example>
+    <param pos="1" name="cookie"/>
+    <param pos="0" name="service.vendor" value="PHP"/>
+    <param pos="0" name="service.family" value="PHP"/>
+    <param pos="0" name="service.product" value="PHP"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:php:php:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(ASPSESSIONID[A-Z]+|ASP\.NET_SessionId|\.ASPXANONYMOUS)=.*">
```
Comment (tsellers-r7) [inline xml/http_cookies.xml:28]: This and above moved up for performance reasons

**Comment by tsellers-r7** [comment]
FYI @hdm

**Comment by hdm** [comment]
Thanks!

**Comment by hdm** [comment]
This LGTM at first glance. We may want to consider normalizing cookies in the future so that we parse each cookie separately vs the entire Set-Cookie header, as the order of cookies can change. Any anchors (^) might not be valid when there is an application that is set first in the set.

**Pull request merged**